WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -429,6 +433,46 @@ extension WordPressAppDelegate {
         WordPressAuthenticator.shared.delegate = authManager
     }
 
+    @available(iOS 13.0, *)
+    func checkAppleIDCredentialState() {
+
+        // If not logged in, remove the Apple User ID from the keychain, if it exists.
+        guard AccountHelper.isLoggedIn else {
+            do {
+                try SFHFKeychainUtils.deleteItem(forUsername: WPAppleIDKeychainUsernameKey,
+                                                 andServiceName: WPAppleIDKeychainUsernameKey)
+            } catch {
+                DDLogDebug("Error while removing Apple User ID: \(error)")
```

Comment:
Do we want to log this as an 'error'? If I'm reading correctly it could simply be that the user has never logged in using SIWA, so there just wouldn't be a keychain item for Apple so we'll be logging out "Error while removing..." on every launch.

Reply:
Hey @frosty .

> user has never logged in using SIWA, so there just wouldn't be a keychain item for Apple so we'll be logging out "Error while removing..." on every launch.

That is correct. That's why I used `DDLogDebug`. But now I see that the actual message can be misleading. I can change it to be less alarming, maybe something like:
`DDLogDebug("Unable to remove Apple User ID from keychain: \(error)")`

I didn't want to _not_ log it since there could be an actual error.